optuna/samplers/_partial_fixed.py

```diff
@@ -0,0 +1,118 @@
+from typing import Any
+from typing import Dict
+import warnings
+
+import numpy
+
+from optuna._experimental import experimental
+from optuna.distributions import BaseDistribution
+from optuna.samplers import BaseSampler
+from optuna.study import Study
+from optuna.trial import FrozenTrial
+
+
+@experimental("2.3.0")
+class PartialFixedSampler(BaseSampler):
+    """Sampler that can sample with some parameters fixed.
+
+        .. versionadded:: 2.3.0
+
+    Example:
+
+        After optimizing with :class:`~optuna.samplers.TPESampler`,
+        fix the value of ``y`` and optimize again with :class:`~optuna.samplers.CmaEsSampler`.
```

Comment:
Could you fix the explanation to align the actual example?
```suggestion
        After several steps of optimization, you can fix the value of ``y`` and re-optimize it.
```